src/console/dbseed/db-seed.service.ts

```diff
@@ -100,33 +103,49 @@ export class DbSeedService {
     }
 
     public readServiceProvider(fileContentAsStr: string): ServiceProvider<true>[] {
-        const serviceProviderFile: EntityFile<ServiceProviderFile> = JSON.parse(
-            fileContentAsStr,
-        ) as EntityFile<ServiceProviderFile>;
-
-        const entities: ServiceProviderFile[] = plainToInstance(ServiceProviderFile, serviceProviderFile.entities);
-
-        const serviceProviders: ServiceProvider<true>[] = entities.map((data: ServiceProviderFile) =>
+        const { entities }: EntityFile<ServiceProvider<true>> = JSON.parse(fileContentAsStr) as EntityFile<
+            ServiceProvider<true>
+        >;
+        const serviceProviders: ServiceProvider<true>[] = entities.map((serviceProviderData: ServiceProvider<true>) =>
             ServiceProvider.construct(
-                data.id,
+                serviceProviderData.id,
                 new Date(),
                 new Date(),
-                data.name,
-                data.url,
-                data.kategorie,
-                data.providedOnSchulstrukturknoten,
-                data.logoBase64 ? Buffer.from(data.logoBase64, 'base64') : undefined,
-                data.logoMimeType,
+                serviceProviderData.name,
+                serviceProviderData.url,
+                serviceProviderData.kategorie,
+                serviceProviderData.providedOnSchulstrukturknoten,
+                serviceProviderData.logo,
```

Comment:
I think this is a mistake, the `service-provider-file.ts` was there for a reason